osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -6,183 +6,234 @@
-                },
-                new Container
+                Type = EdgeEffectType.Shadow,
+                Offset = new Vector2(.250f, 2),
+                Colour = Colour4.Black.Opacity(.25f),
+                Radius = outer_corner_radius
+            };
+
+            {
+                Children = new Drawable[]
                 {
-                    AutoSizeAxes = Axes.Both,
-                    Children = new Drawable[]
+                    backgroundColourBox = new Box
+                    {
+                        Colour = backgroundColour,
+                        RelativeSizeAxes = Axes.Both,
+                        Depth = 2,
+                    },
+                    new Box
+                    {
+                        Alpha = 0.0002f,
+                        Height = 120,
+                        Width = 140,
+                    },
+                    new Container
                     {
-                        ButtonContentContainer = new FillFlowContainer
+                        RelativeSizeAxes = Axes.Both,
+                        Padding = new MarginPadding
                         {
-                            Anchor = Anchor.CentreLeft,
-                            Origin = Anchor.CentreLeft,
-                            Direction = FillDirection.Horizontal,
-                            Shear = -SHEAR,
-                            AutoSizeAxes = Axes.X,
-                            Height = 50,
-                            Spacing = new Vector2(15, 0),
-                            Children = new Drawable[]
+                            Top = 12,
+                            Right = shear_padding,
+                        },
+                        Children = new Drawable[]
+                        {
+                            sprite = new SpriteIcon
                             {
-                                TextContainer = new Container
+                                Shear = -SHEAR,
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                Size = new Vector2(20)
+                            }
+                        }
+                    },
+                    ButtonContentContainer = new FillFlowContainer
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Direction = FillDirection.Horizontal,
+                        AutoSizeAxes = Axes.X,
+                        Children = new Drawable[]
+                        {
+                            TextContainer = new Container
+                            {
+                                Colour = Colour4.White,
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                Shear = -SHEAR,
+                                Margin = new MarginPadding { Top = 41.89f },
```

Comment:
No number should ever be this precise. I don't know how you sourced this but it's not right.

Reply:
Figma

Reply:
@arflyte can you check on this?

Reply:
I check on my side and it says 40, not some floating number.

Reply:
done